Mute ydb/tests/olap/ttl_tiering 3 tests

Mute:<!--mute_list_start-->
ydb/tests/olap/ttl_tiering/sole chunk chunk
ydb/tests/olap/ttl_tiering/ttl_unavailable_s3.py.TestUnavailableS3.test
ydb/tests/olap/ttl_tiering/data_migration_when_alter_ttl.py.TestDataMigrationWhenAlterTtl.test
<!--mute_list_end-->

Branch:<!--branch_list_start-->
main
<!--branch_list_end-->

**Add line to [muted_ya.txt](https://github.com/ydb-platform/ydb/blob/main/.github/config/muted_ya.txt):**
```
ydb/tests/olap/ttl_tiering sole chunk chunk
ydb/tests/olap/ttl_tiering ttl_unavailable_s3.py.TestUnavailableS3.test
ydb/tests/olap/ttl_tiering data_migration_when_alter_ttl.py.TestDataMigrationWhenAlterTtl.test
```

Owner: TEAM:@ydb-platform/cs

**Read more in [mute_rules.md](https://github.com/ydb-platform/ydb/blob/main/.github/config/mute_rules.md)**

**Summary history:** 
 sole chunk chunk: Flaky last 2 days, at 2025-01-23: success_rate 78%, Pass:47 Fail:13 Mute:0 Skip:0
ttl_unavailable_s3.py.TestUnavailableS3.test: Flaky last 1 days, at 2025-01-23: success_rate 50%, Pass:3 Fail:3 Mute:0 Skip:0
data_migration_when_alter_ttl.py.TestDataMigrationWhenAlterTtl.test: Flaky last 2 days, at 2025-01-23: success_rate 69%, Pass:30 Fail:13 Mute:0 Skip:0


**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?full_name=__in_ydb%2Ftests%2Folap%2Fttl_tiering%2Fsole+chunk+chunk&full_name=__in_ydb%2Ftests%2Folap%2Fttl_tiering%2Fttl_unavailable_s3.py.TestUnavailableS3.test&full_name=__in_ydb%2Ftests%2Folap%2Fttl_tiering%2Fdata_migration_when_alter_ttl.py.TestDataMigrationWhenAlterTtl.test)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

```
ydb/tests/olap/ttl_tiering/data_migration_when_alter_ttl.py:203: in test
    raise Exception("Bucket1 is not empty")
E   Exception: Bucket1 is not empty
```

--- a/.github/config/muted_ya.txt
+++ b/.github/config/muted_ya.txt
@@ -176,11 +176,6 @@ ydb/tests/olap/scenario test_alter_tiering.py.TestAlterTiering.test[many_tables]
 ydb/tests/olap/scenario test_insert.py.TestInsert.test[read_data_during_bulk_upsert]
 ydb/tests/olap/scenario test_insert.py.TestInsert.test_multi[read_data_during_bulk_upsert]
 ydb/tests/olap/scenario test_read_update_write_load.py.TestReadUpdateWriteLoad.test[read_update_write_load]
-ydb/tests/olap/ttl_tiering [*/*] chunk chunk
-ydb/tests/olap/ttl_tiering [data_migration_when_alter_ttl.py] chunk chunk
-ydb/tests/olap/ttl_tiering [ttl_delete_s3.py] chunk chunk
-ydb/tests/olap/ttl_tiering data_migration_when_alter_ttl.py.TestDataMigrationWhenAlterTtl.test
-ydb/tests/olap/ttl_tiering sole chunk chunk
 ydb/tests/postgres_integrations/go-libpq docker_wrapper_test.py.test_pg_generated[Test64BitErrorChecking]
 ydb/tests/postgres_integrations/go-libpq docker_wrapper_test.py.test_pg_generated[TestAppendEncodedText]
 ydb/tests/postgres_integrations/go-libpq docker_wrapper_test.py.test_pg_generated[TestAppendEscapedTextExistingBuffer]

